prow/cmd/peribolos/main.go

```diff
@@ -409,8 +415,11 @@ func configureTeams(client teamClient, orgName string, orgConfig org.Config) (ma
 	}
 
 	// Delete undeclared teams.
-	// TODO(fejta): consider ensuring we don't delete too many teams at once
-	unused := ints.Difference(used)
+	if reused := unused.Intersection(used); len(reused) > 0 {
```

Comment:
s/unused/ints/

Reply:
No? This checks if the actually used team IDs overlap with the ones expected to be unused. The variable names could probably be better.